Tests/SonarScanner.MSBuild.TFS.Tests/ProgramTests.cs

```diff
@@ -0,0 +1,123 @@
+﻿/*
+ * SonarScanner for MSBuild
+ * Copyright (C) 2016-2019 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.IO;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using TestUtilities;
+
+namespace SonarScanner.MSBuild.TFS.Classic.Tests
+{
+    [TestClass]
+    public class ProgramTests
+    {
+        [TestMethod]
+        public void Execute_WhenNoArgs_ShouldLogError()
+        {
+            var logger = new TestLogger();
+            var result = Program.Execute(new string[] { }, logger);
+
+            result.Should().Be(0);
+            logger.Warnings.Should().BeEmpty();
+            logger.Errors.Should().HaveCount(1);
+            logger.Errors.Should().Contain(new string[] { "No argument found. Exiting..." });
+        }
+
+        [TestMethod]
+        public void Execute_Method_Is_Uknown_Should_Log_Error()
+        {
+            var logger = new TestLogger();
+            var result = Program.Execute(new string[] { "MockMethod" }, logger);
+
+            result.Should().Be(0);
+            logger.Errors.Should().HaveCount(2);
+            logger.Errors.Should().Contain(new string[] { "An exception occured while executing the process : Requested value 'MockMethod' was not found." });
+        }
+
+        [TestMethod]
+        public void Execute_ShouldExecute_CoverageConverter_ShouldSucceeed()
+        {
+            var logger = new TestLogger();
+            var tempDir = Directory.CreateDirectory(Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString())).FullName;
+            var text = "<?xml version=\"1.0\" encoding=\"utf-8\"?>" +
```

Comment:
No need to change this, but the XML would be more readable if you just made it a multi-line string i.e. `var text = @"..."`

You can also combine multiline and interpolated strings if necessary using `var text = $@"...  "`